VideoToImage/README.md

```diff
@@ -0,0 +1,12 @@
+Chops video into images
```

Comment:
can you format this to have some headers and maybe some bullet points? Markdown is better used if you do some simple formatting.